app/controllers/api/scores_controller.rb

```diff
@@ -4,8 +4,24 @@ class ScoresController < ApplicationController
     before_action :logged_in!
     before_action :validate_score_user_id, only: :destroy
 
+    def show_scores
+      user = User.find_by(id: params[:id])
+      if user
+        scores = (user.scores.order(played_at: :desc, id: :desc).includes(:user)).map(&:serialize)
```

Comment:
Why do you need round brackets around `user.scores.order(played_at: :desc, id: :desc).includes(:user))`?